test/tests/userobjects/Terminator/tests

```diff
@@ -25,7 +38,8 @@
   []
   [terminator_message_2]
     type = 'RunException'
-    expect_err = "Setting the error level to ERROR always causes a hard failure, which is incompatible with `fail_mode=SOFT`."
+    expect_err = "Setting the error level to ERROR always causes a hard failure, which is "
+                 "incompatible with `fail_mode=SOFT`."
```

Comment:
```suggestion
    expect_err = "Setting the error level to error always causes a hard failure, which is "
                 "incompatible with a soft failure mode."
```
Here I am introducing my own redundancy but I don't really know how not to

Reply:
ERROR and fail_mode=SOFT are in the parameters.
I think it s clearer how it used to be.

undid the change

Reply:
Requirements are supposed to be independent of implementation. The requirement is conceptually supposed to occur before the implementation. This concretely shows implementation

Reply:
This isnt a requirement though. It s an expected error message

Reply:
Oh my goodness. I'm a total dummy. I'm blind as a bat!